### PR TITLE
kwarg-ify methods

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py [hunk 1]
```diff
@@ -256,7 +248,7 @@ def __init__(
         return await upload_append_blob(**options)
 
     @distributed_trace_async
-    async def download_blob(self, offset=None, length=None, validate_content=False, **kwargs):
+    async def download_blob(self, offset=None, length=None, **kwargs):
```
Comment (lmazuel) [inline sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py]: Remove a bool from #type

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_service_client_async.py [hunk 2]
```diff
@@ -169,8 +167,8 @@ def __init__(
             process_storage_error(error)
 
     @distributed_trace_async
-    async def get_service_stats(self, timeout=None, **kwargs): # type: ignore
-        # type: (Optional[int], **Any) -> Dict[str, Any]
+    async def get_service_stats(self, **kwargs): # type: ignore
+        # type: (**Any) -> Dict[str, Any]
```
Comment (lmazuel) [inline sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_service_client_async.py]: **Any should be just Any

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py [hunk 3]
```diff
@@ -138,11 +136,8 @@ def __init__(
     async def upload_blob(
             self, data,  # type: Union[Iterable[AnyStr], IO[AnyStr]]
             blob_type=BlobType.BlockBlob,  # type: Union[str, BlobType]
-            overwrite=False,  # type: bool
```
Comment (johanste) [inline sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py]: Please open an issue to change the :param: directive for the keyword parameter to :keyword: in the docstring.

File: sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py [hunk 4]
```diff
@@ -443,11 +445,8 @@ def _upload_blob_options(  # pylint:disable=too-many-statements
     def upload_blob(  # pylint: disable=too-many-locals
             self, data,  # type: Union[Iterable[AnyStr], IO[AnyStr]]
             blob_type=BlobType.BlockBlob,  # type: Union[str, BlobType]
-            overwrite=False,  # type: bool
             length=None,  # type: Optional[int]
             metadata=None,  # type: Optional[Dict[str, str]]
-            content_settings=None,  # type: Optional[ContentSettings]
-            validate_content=False,  # type: Optional[bool]
```
Comment (johanste) [inline sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py]: should `max_concurrency` also be a `**kwarg`?

File: sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py (hunk 3 above)
Comment (rakshith91) [reply sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py]: https://github.com/Azure/azure-sdk-for-python/issues/7619

File: sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py (hunk 4 above)
Comment (rakshith91) [reply sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py]: i'm not entirely in favor of it..it's something that should be used for optimal performance,

**Comment by rakshith91** [comment]
no idea why python - loganalytics - ci  and python - machinelearning - ci are triggered

**Pull request merged**